### PR TITLE
Heap is not thread safe in scheduling queue

### PR DESCRIPTION
**Pull request by CaoShuFeng**
/cc @bsalamat 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by bsalamat** [comment]
Thanks, @CaoShuFeng for fixing the comments.

/lgtm

File: plugin/pkg/scheduler/core/scheduling_queue.go [hunk 1]
```diff
@@ -149,7 +149,7 @@ type UnschedulablePods interface {
 // scheduling. This is called activeQ and is a Heap. Another queue holds
 // pods that are already tried and are determined to be unschedulable. The latter
 // is called unschedulableQ.
-// Heap is already thread safe, but we need to acquire another lock here to ensure
+// Heap is not thread safe, so we need to acquire a lock here to ensure
```
Comment (bsalamat) [inline plugin/pkg/scheduler/core/scheduling_queue.go]: We could just remove this line and the one below. I was initially using a thread safe heap and that's why I was trying to explain why we needed to acquire another lock.

Comment (CaoShuFeng) [reply plugin/pkg/scheduler/core/scheduling_queue.go]: Done.

**Pull request merged**